packages/transaction-controller/src/TransactionController.test.ts

```diff
@@ -86,7 +86,7 @@ import {
   getTransactionLayer1GasFee,
   updateTransactionLayer1GasFee,
 } from './utils/layer1-gas-fee-flow';
-import { shouldResimulate } from './utils/resimulate';
+import { shouldResimulate } from './helpers/ResimulateHelper';
```

Comment:
In order to keep the list of imports sorted, this line should be moved up to go below the following line:

``` typescript
import { PendingTransactionTracker } from './helpers/PendingTransactionTracker';
```